src/PythonMigrationViewExtension/Differ/IDiffViewViewModel.cs

```diff
@@ -1,9 +1,40 @@
-﻿namespace Dynamo.PythonMigration.Differ
+﻿using System;
+using System.Windows;
+using System.Windows.Data;
+
+namespace Dynamo.PythonMigration.Differ
 {
+    public enum State
+    {
+        NoChanges,
+        HasChanges,
+        Error
+    }
+
+    public class DiffStateToVisibilityConverter : IValueConverter
```

Comment:
Maybe this can be moved out to its own file since it's not part of the interface itself.